pkg/web/web.go

```diff
@@ -34,6 +34,7 @@ func RunHTTPServer() {
 	router := mux.NewRouter()
 	apiV1 := router.PathPrefix("/v1.0").Subrouter()
 	apiV1.HandleFunc("/migrate-machine", GenericResponseHandler(MigrateMachine)).Methods("POST")
+	apiV1.HandleFunc("/container/delete", GenericResponseHandler(DestroyContainer)).Methods("POST")
```

Comment:
instead of ```/container/delete``` I suggest to use ```destroy-container```, in the nearest future I would like to use JSON-RPC v2.0